generators/client/templates/react/package.json.ejs

```diff
@@ -99,29 +99,29 @@ limitations under the License.
     "chai": "4.2.0",
     "chai-as-promised": "7.1.1",
     <%_ } _%>
-    "copy-webpack-plugin": "5.1.0",
-    "core-js": "3.4.8",
-    "cross-env": "6.0.3",
-    "css-loader": "3.3.0",
-    "enzyme": "3.10.0",
-    "enzyme-adapter-react-16": "1.15.1",
-    "enzyme-to-json": "3.4.3",
-    "eslint": "6.7.2",
+    "copy-webpack-plugin": "5.1.1",
+    "core-js": "3.6.5",
+    "cross-env": "7.0.2",
+    "css-loader": "3.5.3",
+    "enzyme": "3.11.0",
+    "enzyme-adapter-react-16": "1.15.2",
+    "enzyme-to-json": "3.4.4",
+    "eslint": "6.8.0",
     "eslint-config-jhipster": "0.0.1",
     "eslint-config-jhipster-react": "0.0.1",
-    "eslint-config-prettier": "6.7.0",
-    "eslint-loader": "3.0.3",
-    "eslint-plugin-react": "7.17.0",
-    "file-loader": "5.0.2",
-    "fork-ts-checker-webpack-plugin": "3.1.1",
+    "eslint-config-prettier": "6.11.0",
+    "eslint-loader": "4.0.2",
+    "eslint-plugin-react": "7.19.0",
+    "file-loader": "6.0.0",
+    "fork-ts-checker-webpack-plugin": "4.1.3",
     "friendly-errors-webpack-plugin": "1.7.0",
     "generator-jhipster": "<%= packagejs.version %>",
     "html-webpack-plugin": "3.2.0",
     <%_ if (!skipCommitHook) { _%>
-    "husky": "3.1.0",
+    "husky": "4.2.5",
```

Comment:
FYI this will conflict with https://github.com/jhipster/generator-jhipster/pull/11645/files where I introduce a constant to use the same version for angular, react and microservice.

Reply:
we have to update depending on which PR is merged first